logistics_project/apps/malawi/warehouse/runner.py

```diff
@@ -497,10 +497,15 @@ def update_consumption_times(since):
     """
     Update the consumption time_with_data values for any supply point / product pairings
     where the supply point doesn't explicitly supply the product.
+    This function only works against base level data.
     """
     # any consumption value that was touched potentially needs to have its
     # time_needing_data updated
-    consumptions_to_update = CalculatedConsumption.objects.filter(update_date__gte=since)
+    consumptions_to_update = CalculatedConsumption.objects.filter(
+        Q(product__type__base_level=BaseLevel.HSA, supply_point__type__code=SupplyPointCodes.HSA) |
+        Q(product__type__base_level=BaseLevel.FACILITY, supply_point__type__code=SupplyPointCodes.FACILITY),
+        update_date__gte=since
+    )
```

Comment:
hey, that sounds right to me. based on where this is called in the normal warehouse workflow I think maybe it works fine because the only possible `CalculatedConsumption`s that would be updated since the warehouse started are the base level ones, yeah? (assuming nothing else is touching them concurrently). Either way being explicit about it and not assuming anything about the context around which it is called makes sense to me.